starknet/README.md

```diff
@@ -0,0 +1,31 @@
+---
+namespace-identifier: starknet
+title: StarkNet Ecosystem
+author: Argent Labs
+discussions-to:
+status: Draft
+type: Informational
+created: 2022-12-04
+updated: 2022-12-04
+replaces: CAIP-28
+---
+
+# Namespace for StarkNet
+
+## Introduction
+
+This document describes the syntax and structure of the namespace for StarkNet..
+
+## Syntax
+
+The namespace "starknet" refers to the StarkNet Layer 2 chain on Ethereum.
+
+## References
+
+[Type system]: https://starknet.io/docs/reference/syntax.html#type-system
```

Comment:
Cairo types seem like a very low-level link in this context, and will also stop being relevant with Cairo 1.0, so I think that we can remove this.